charts/k8s-monitoring/values.yaml

```diff
@@ -403,6 +403,78 @@ metrics:
     # -- The list of API Server metrics that will be scraped by the Agent
     allowList: []
 
+  # Metrics from the Kube Controller Manager
+  kubeControllerManager:
+    # -- Scrape metrics from the Kube Controller Manager
+    enabled: false
+
+    # -- (int) Port number used by the Kube Controller Manager, set by --secure-port.
+    port: 10257
+
+    # -- (string) How frequently to scrape metrics from the Kube Controller Manager
+    # Overrides metrics.scrapeInterval
+    # @default -- 60s
+    scrapeInterval:
+
+    # -- (string) Rule blocks to be added to the discovery.relabel component for the Kube Controller Manager.
+    # See https://grafana.com/docs/agent/latest/flow/reference/components/discovery.relabel/#rule-block
+    extraRelabelingRules:
+
+    # -- (string) Rule blocks to be added to the prometheus.relabel component for the Kube Controller Manager.
+    # See https://grafana.com/docs/agent/latest/flow/reference/components/prometheus.relabel/#rule-block
+    extraMetricRelabelingRules:
+
+    # -- The list of Kube Controller Manager metrics that will be scraped by the Agent
+    allowList: []
+
+  # Metrics from the Kube Proxy
+  kubeProxy:
+    # -- Scrape metrics from the Kube Proxy
+    enabled: false
+
+    # -- (int) Port number used by the Kube Proxy, set by --secure-port.
```

Comment:
I think kube-proxy doesn't have `--secure-port`, it has `--metrics-bind-address` and it is HTTP only (not HTTPS). Also the value for this arg needs to contain both the IP and the port like `0.0.0.0:10249`.

Here's [the doc](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-proxy/) with the CLI args and I saw scrape failures when I tried https (as the scheme was not documented). It's inconsistent with controller manager and scheduler for some reason.

Looks like you have the correct config in `.github/configs/cluster-config.yaml` but not sure if you need to specify the http scheme maybe in the scrape component.

Reply:
Scraping is done by http by default. the `prometheus.scrape "kube_proxy"` component does not set https like the other ones do. But I should fix the comment.